stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -133,6 +133,7 @@ public class MessageInputView : ConstraintLayout {
 
     public fun setCommands(commands: List<Command>) {
         suggestionListController?.commands = commands
+        setCommandsEnabled(commands.isNotEmpty())
```

Comment:
This `setCommandsEnabled()` call could override a previous `setCommandsEnable(false)` that customer used, because this method is called from a livedata subscription (We don't know when data will arrive)
What about providing some property from `suggestionListControler` that tell us if "inside commands behavior" can be used and checking it from the `refreshControlsState()` method? :thinking:

Reply:
🙈 I totally forget about that. Thanks one more time!